app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -372,8 +364,8 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
         }
 
         navUrlInput.nextFocusDownId = when {
-            pocketVideosContainer.isEffectivelyVisible -> R.id.pocketVideoMegaTileView
-            megaTileTryAgainButton.isEffectivelyVisible -> R.id.megaTileTryAgainButton
+            pocketViewModel.state.value is PocketViewModel.State.Feed -> R.id.pocketVideoMegaTileView
```

Comment:
nit: why is one reference equality expression use `is` while the other use `===`?

Reply:
Yeah, that's annoying and I'm not a big fan of it.  `Feed` is a data class (it carries video information) whereas `Error` is an object, so testing its type and testing referential equality are the same.  Testing `Error` with an `is` results in a linting error.  I'm not sure what the best way to approach this is, tbh.